ceph-salt-formula/salt/ceph-salt/cephbootstrap.sls

```diff
@@ -70,12 +75,13 @@ configure ssh orchestrator:
         ceph config-key set mgr/cephadm/ssh_identity_pub -i /tmp/ceph-salt-ssh-id_rsa.pub
         ceph mgr module enable cephadm && \
         ceph orch set backend cephadm
```

Comment:
@tserong do we also need to tun `ceph orch apply crash` here?

Reply:
> @tserong do we also need to tun `ceph orch apply crash` here?

I don't think so. We *do* ultimately need `ceph orch apply crash` to happen in the upgrade case, but I don't think this is the right place for it. We actually need to apply service specs for all adopted services, and can probably apply the crash service when doing that, but this is outside the scope of what ceph-salt does so far (if that changes, we can always add it in later)